ci/test.sh

```diff
@@ -33,6 +33,49 @@ convox rack logs --no-follow --since 30m
 convox rack logs --no-follow --since 30m | grep service/
 convox rack ps | grep system
 
+# downgrade persistance testing
+if [ "$1" == "downgrade-setup" ]; then
+    # app (httpd2)
+    cd $root/examples/httpd
+    convox apps create httpd2
+    convox apps
+    convox apps | grep httpd2
+    convox apps info httpd2 | grep running
+    convox deploy -a httpd2
+    endpoint=$(convox api get /apps/httpd2/services | jq -r '.[] | select(.name == "web") | .domain')
+    fetch https://$endpoint | grep "It works"
+    echo "ENDPOINT=${endpoint}" | convox env set -a httpd2
+    # postgres resource test
+    convox resources -a httpd2 | grep postgresdb
+    ps=$(convox api get /apps/httpd2/processes | jq -r '.[]|select(.status=="running" and .name == "resource-tester")|.id' | head -n 1)
+    convox exec $ps "/usr/scripts/db_insert.sh" -a httpd2
+    convox exec $ps "/usr/scripts/db_check.sh" -a httpd2
+    # redis resource test
+    convox resources -a httpd2 | grep rediscache
+    ps=$(convox api get /apps/httpd2/processes | jq -r '.[]|select(.status=="running" and .name == "resource-tester")|.id' | head -n 1)
+    convox exec $ps "/usr/scripts/redis_check.sh" -a httpd2
+
+    # downgrate-setup runs before the downgrading, so no need to test others functionality
+    # they are covered by other ci test
+    exit 0
```

Comment:
Creating another shell script will make this clear.

Reply:
Using a different shell script might be a good option since we'll also be adding upgrade-setup and upgrade-check